DBMS/DBMS-Session2-Assignment_1/Assignment_1.txt

```diff
@@ -0,0 +1,111 @@
+DROP DATABASE shopcart;
+CREATE DATABASE ShopCart;
+SHOW DATABASES;
+USE ShopCart;
+SHOW tables;
+
+#create CATEGORY table-----------------
+CREATE TABLE category(
+    category_id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    category_title VARCHAR(20) NOT NULL,
+    category_desc VARCHAR(100) NOT NULL,
+    parent_category_id INT,
+    FOREIGN KEY(parent_category_id) REFERENCES category(category_id)
+);
+
+#create PRODUCT table-------------------
+CREATE TABLE product(
+    product_id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    product_title VARCHAR(50) NOT NULL,
+    product_desc VARCHAR(100) NOT NULL,
+    product_date DATE NOT NULL,
+    product_price DOUBLE NOT NULL,
+    product_quantity INT NOT NULL,
```

Comment:
Why all these fields required ?

Reply:
product date use for product added on that date
product_price - unit price of a product
product_quantity - total stock quantity
@metaamitasharma